package.json

```diff
@@ -2,56 +2,48 @@
   "name": "@loke/http-rpc-client",
   "version": "4.1.1",
   "description": "",
-  "main": "./dist/index.js",
-  "types": "./dist/index.d.ts",
+  "main": "./dist/index.cjs",
+  "module": "./dist/index.mjs",
+  "types": "./dist/index.d.cts",
   "files": [
     "dist"
   ],
+  "exports": {
+    "require": {
+      "types": "./dist/index.d.cts",
+      "default": "./dist/index.cjs"
+    },
+    "import": {
+      "types": "./dist/index.d.mts",
+      "default": "./dist/index.mjs"
+    }
+  },
   "scripts": {
-    "fix": "eslint . --fix",
-    "lint": "eslint .",
-    "posttest": "npm run lint",
-    "test": "ava -v",
-    "prepare": "npm run build",
-    "build": "rm -rf ./dist && tsc"
+    "test": "tsx --test ./src/*.test.ts",
+    "lint": "eslint . && prettier --check .",
+    "build": "pkgroll --sourcemap",
+    "prepare": "npm run build"
   },
   "author": "Dominic Smith",
   "license": "ISC",
-  "ava": {
-    "files": [
-      "test/**/*"
-    ],
-    "extensions": [
-      "ts",
-      "js"
-    ],
-    "require": [
-      "ts-node/register"
-    ]
+  "engines": {
+    "node": ">=20"
   },
   "devDependencies": {
-    "@types/body-parser": "^1.19.2",
-    "@types/node": "^15.14.9",
-    "@types/node-fetch": "^2.6.2",
-    "ava": "^3.15.0",
-    "body-parser": "^1.20.2",
-    "eslint": "^7.27.0",
-    "eslint-config-prettier": "^8.3.0",
-    "eslint-plugin-prettier": "^3.4.0",
-    "express": "^4.15.2",
-    "pify": "^2.3.0",
-    "prettier": "^2.3.0",
-    "prom-client": "^14.1.1",
-    "ts-node": "^10.9.1",
-    "typescript": "^4.8.2"
+    "@eslint/js": "^9.18.0",
+    "@tsconfig/node20": "^20.1.4",
+    "@types/node": "^20.17.17",
+    "eslint": "^9.18.0",
+    "eslint-config-prettier": "^10.0.1",
+    "pkgroll": "^2.6.1",
+    "prettier": "3.5.0",
+    "prom-client": "^15.1.3",
+    "tsx": "^4.19.2",
+    "typescript": "^5.7.3",
+    "typescript-eslint": "^8.21.0"
   },
   "dependencies": {
-    "@loke/context": "^0.0.1",
-    "find-up": "^1.1.2",
-    "got": "^6.3.0",
-    "node-abort-controller": "^3.0.1",
-    "node-fetch": "2.6.7",
-    "p-finally": "^1.0.0"
+    "@loke/context": "^0.0.1"
```

Comment:
Nice, almost 0 deps. 
is it worth having @loke/context as a peer dep?
Or it is small enough to not worry.

Reply:
It might need to be a peer dep, but for for now there is only one version so can deal with that when its an issue